app/assets/stylesheets/administrate/base/_forms.scss

```diff
@@ -1,78 +1,99 @@
 fieldset {
-  background: lighten($base-border-color, 10);
-  border: $base-border;
-  margin: 0 0 ($base-spacing / 2) 0;
-  padding: $base-spacing;
+  background-color: transparent;
+  border: 0;
+  margin: 0;
+  padding: 0;
 }
 
-input,
-label,
-select {
-  display: block;
-  font-family: $form-font-family;
-  font-size: $form-font-size;
+legend {
+  font-weight: 600;
+  margin-bottom: $small-spacing / 2;
+  padding: 0;
 }
 
 label {
-  font-weight: bold;
-  margin-bottom: $base-spacing / 4;
-
-  &.required:after {
-    content: "*";
-  }
+  display: block;
+  font-weight: 600;
+  margin-bottom: $small-spacing / 2;
+}
 
-  abbr {
-    display: none;
-  }
+input,
+select {
+  display: block;
+  font-family: $base-font-family;
+  font-size: $base-font-size;
 }
 
+[type="color"],
```

Comment:
I bet Mr. Hound is talking about Line `44` (`input:not([type])`), which is an edge case and deserves to be there. But I don’t want to whole-hog turn this linter off.